src/utils.ts

```diff
@@ -0,0 +1,59 @@
+export function getFirstDayOfTheWeek(date: Date): Date {
+  let newDate = new Date(date);
+  const day = newDate.getDay();
+  const diff = newDate.getDate() - day + (day === 0 ? -6 : 1);
+  const startOfWeek = new Date(newDate.setDate(diff));
+  startOfWeek.setHours(0, 0, 0, 0);
+  return startOfWeek;
+}
+
+export function isToday(date: Date): boolean {
+  const today = new Date();
+  return (
+    date.getDate() === today.getDate() &&
+    date.getMonth() === today.getMonth() &&
+    date.getFullYear() === today.getFullYear()
+  );
+}
+
+export const getDisplayedMonth = (date: Date) => {
+  return new Date(date.getFullYear(), date.getMonth(), 1);
+};
+
+export const getFirstDayOfMonth = (date: Date) => {
+  return new Date(date.getFullYear(), date.getMonth(), 1);
+};
```

Comment:
Either naming or implementation is not correct. : )
From first function naming I would expect to get either number that represents month, eg 7, or string that represents month 'July'. Same with the second one.